index.html

```diff
@@ -37,8 +37,8 @@ <h4 class="socials">LETs CONNECT</h4>
             <li><a href="#"><img class="social-media" src="img/github.png" alt="github"></a></li>
             <li><a href="#"><img class="social-media" src="img/vec.png" alt="vector"></a></li>
         </ul>
-
     </section>
+
     <section class="works">
```

Comment:
- [ ] There is a huge empty space that I am seeing from the top heading/ landing section and the works section which should not be here. Kindly look into this.

**This is the empty space I am talking about:**

![image](https://user-images.githubusercontent.com/86318284/185758435-9ef60f69-b997-4c4a-8f25-c0a60bb89122.png)


- [ ] I am only able to see one "projects" card in the section of your work. Kindly incldde the other two "projects" as they are in the chosen Figma design.